i3pystatus/core/modules.py

```diff
@@ -13,6 +13,7 @@ class Module(SettingsBase):
         ('on_rightclick', "Callback called on right click (see :ref:`callbacks`)"),
         ('on_upscroll', "Callback called on scrolling up (see :ref:`callbacks`)"),
         ('on_downscroll', "Callback called on scrolling down (see :ref:`callbacks`)"),
+        ('hints', "Additional output blocks for module output (see :ref:`hints`)"),
```

Comment:
These refferences are not displaying correctly in the [documentation](http://docs.enkore.de/i3pystatus/i3pystatus.core.html#i3pystatus.core.modules.IntervalModule.settings).